jta-1_2-in-wildfly/pom.xml

```diff
@@ -145,7 +126,9 @@
         <profile>
             <id>arq</id>
             <activation>
-                <activeByDefault>true</activeByDefault>
+                <property>
+                    <name>!ipv6</name>
+                </property>
```

Comment:
Same here (even though in this particular case, it makes more sense to connect `ipv6` to arquillian with or without IPv6

Reply:
Thanks @jmfinelli ! This is also my mistake, I will restore the original version of the profile configuration.